suite-etl/pipelines/raw/transactions.py

```
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+"""
+Transactions Table Handler
+
+This module provides functions for interacting with the transactions table:
+- Loading transaction data
+- Processing transactions
+"""
+
+import pyarrow as pa
```

Ruff (0.11.9)

`pyarrow` imported but unused

Remove unused import: `pyarrow`

(F401)